docs/spec/resolution.md

```diff
@@ -84,7 +84,7 @@
                 1. Set the _Next Update Commitment_ value to `null`.
                 2. Exit [update operation processing](#update-operation-processing) and advance to [compiled state processing](#compiled-state-processing).
         3. Validate the [_Chunk File Delta Entry_](#chunk-file-delta-entry). If the [_Chunk File Delta Entry_](#chunk-file-delta-entry) is valid, proceed, if the entry is invalid, let the state of the DID reflect an _Empty DID State_.
-        4. Generate a hash of the decoded [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [Hashing Process](#hashing-process) and ensure the hash equals the value of the [_Update Operation Signed Data Object_](#update-signed-data-object) `delta_hash` property. If the values are exactly equal, proceed, if they are not:
+        4. Generate a hash of the canonicalized [_Chunk File Delta Entry_](#chunk-file-delta-entry) via the [`HASH_PROTOCOL`](#hash-protocol) and ensure the hash equals the value of the [_Update Operation Signed Data Object_](#update-signed-data-object) `delta_hash` property. If the values are exactly equal, proceed, if they are not:
```

Comment:
compression question again.

Reply:
Took care of this with added processing step.